### DISCUSSION TITLE
New Experience: Intro to Open Source

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#119982] [GitHub Education]
![repo](https://github.com/community/community/assets/107881423/45a9881c-9685-489d-b7ab-4246c1e8f276)

The open source ecosystem can sometimes be intimidating to enter if you’re a new developer or just getting started contributing to projects. We want to make your journey getting started with open source as smooth as we can. That’s why we’ve partnered with OpenSauced to bring you our newest learning Experience: Intro to Open Source. It’s a curated resource hub with the tools, articles, and videos you need to break into open source and start building a community. Here's how to get started with the [Intro to Open Source Experience](https://education.github.com/experiences/intro_to_open_source).

![Orange and Black](https://github.com/community/community/assets/107881423/4ac9efae-269d-4943-be1a-f129b5b16da9)

**Step 1: Start with the OpenSauced [Intro to Open Source course](https://github.com/open-sauced/intro). It’s the best way to gain an overview of what open source is, how to contribute and what tools you’ll need to get started.**

![beginners](https://github.com/community/community/assets/107881423/5d65dae0-c35e-429b-9448-6b82deb429a1)

**Step 2: Bookmark these [Open Source Guides](https://opensource.guide/), a great resource to return to throughout your journey.**

![oss-bike](https://github.com/community/community/assets/107881423/2bde40a8-72d5-4342-8c0b-786bacf32838)

**Step 3: Leverage the amazing work from GitHub’s own DevRel team by absorbing some of their foundational content on open source including:** 

- This article series from @ MishManners
- Gaining knowledge on open source vocabulary with these handy videos from GitHub
- [Part 1](https://www.youtube.com/watch?v=ViC0j7jjKnU), [Part 2](https://www.youtube.com/watch?v=kIMNgGfwvMA)
- Exploring [Open Source Fridays](https://opensourcefriday.com/)

![contribute](https://github.com/community/community/assets/107881423/819bbd68-45b6-4d3c-bd9c-2ee4e2803c3e)

**Step 4: Start contributing and building community.**
Explore [Good First Issues](https://forgoodfirstissue.github.com/) and lend your skills to an open source project focused on the [Digital Public Goods (DPGs)](https://digitalpublicgoods.net/digital-public-goods/). 
[Building Communities](https://docs.github.com/en/communities) docs

Finally - If you’re a verified student, you also have access to Student Developer Pack partners whose products are perfect for learning about open source and collaborating with others. Take Codédex’s Git and GitHub courses, and explore offers from VScode, GitLens, GitKraken to help you along your way.

Go to the [Experience page](https://education.github.com/experiences/intro_to_open_source) and get started today!

**Comment by dwaynesenpai25** [comment]
Its great and gives me an amazing experience of the basic steps in start using github
